README.rst

```diff
@@ -2,102 +2,77 @@
 conda-devenv
 ============
 
-
-.. image:: https://img.shields.io/pypi/v/conda-devenv.svg
-        :target: https://pypi.python.org/pypi/conda-devenv
-
 .. image:: https://img.shields.io/travis/ESSS/conda-devenv.svg
-        :target: https://travis-ci.org/ESSS/conda-devenv
+    :target: https://travis-ci.org/ESSS/conda-devenv
 
 .. image:: https://readthedocs.org/projects/conda-devenv/badge/?version=latest
-        :target: https://conda-devenv.readthedocs.io/en/latest/?badge=latest
-        :alt: Documentation Status
+    :target: https://conda-devenv.readthedocs.io/en/latest/?badge=latest
+    :alt: Documentation Status
 
-.. image:: https://pyup.io/repos/github/esss/conda-devenv/shield.svg
-     :target: https://pyup.io/repos/github/esss/conda-devenv/
-     :alt: Updates
+.. image:: https://anaconda.org/conda-forge/conda-devenv/badges/version.svg
+    :target: https://anaconda.org/conda-forge/conda-devenv
 
+``conda-devenv`` is ``conda`` extension to work with multiple projects in development mode.
 
-A tool to work with multiple projects in develop-mode using conda-env.
+It works by processing ``environment.devenv.yml`` files, similar to how ``conda env``
+processes ``environment.yml`` files, with this additional features:
 
-.. code-block:: console
+* `Jinja 2 <http://jinja.pocoo.org/docs/2.9/>`_ support: gives more flexibility to the environment
+  definition, for example making it simple to conditionally add dependencies based on platform.
+
+* ``include`` other ``environment.devenv.yml`` files: this allows you to easily work in several
+  dependent projects at the same time, managing a single ``conda`` environment with your dependencies.
+
+* **Environment variables**: you can define a ``environment:`` section with environment variables
+  that should be defined when the environment is activated.
 
-    $ cat environment.devenv.yml
-    {% set CONDA_PY = os.environ.get('CONDA_PY', '27') %}
-    name: awesome-project-py{{ CONDA_PY }}
+Here's a simple ``environment.devenv.yml`` file:
+
+.. code-block:: yaml
+
+    {% set conda_py = os.environ.get('CONDA_PY', '35') %}
+    name: web-ui-py{{ conda_py }}
 
     includes:
-      - {{ root }}/../other-project/environment.devenv.yml
+      - {{ root }}/../core-business/environment.devenv.yml
```

Comment:
Maybe (to keep it short):

web-ui/environment.devenv.yml:
```yaml
 {% set conda_py = os.environ.get('CONDA_PY', '35') %}
name: web-ui-py{{ conda_py }}
includes:
  - {{ root }}/../core-business/environment.devenv.yml
environment:
  DATAS:
    - {{ root }}/datas
```
core-business/environment.devenv.yml:
```yaml
 {% set conda_py = os.environ.get('CONDA_PY', '35') %}
name: core-business-py{{ conda_py }}
dependencies:
  {% if sys.platform.startswith('linux') %}
  - ccache
  {% endif %}
environment:
  DATAS:
    - {{ root }}/datas
```

```console
     $ cd ~/projects/web-ui
     $ conda devenv
     > Executing: conda env update --file environment.yml --prune
     Fetching package metadata .........
     Solving package specifications: ..........
     Linking packages ...
    [      COMPLETE      ]|############################| 100%
     #
     #
     # To activate this environment, use:
     # To activate this environment, use:
     # > source activate web-ui-py35
     #
     #
     $ source activate web-ui-py35
 
     $ env DATAS
     /home/user/projects/web-ui/datas:/home/user/projects/core-business/datas
```